content/docs/get-started/aws/review-project.md

```diff
@@ -82,16 +82,31 @@ class Program
 This Pulumi program creates an S3 bucket and exports the name of the bucket.
 
 {{% lang python %}}
+
+> *Note*: As a prerequisite, [install virtualenv](https://virtualenv.pypa.io/en/latest/installation/) to manage project requirements
+
 For Python, before we deploy the stack, the following commands need to be run to create a virtual environment, activate it, and install dependencies:
 
+Create a virtual environment:
+
 ```bash
-$ virtualenv -p python3 venv
+$ python -m venv aws-env
 ```
 
+Activate the environment in Linux and macOS:
+
 ```bash
-$ source venv/bin/activate
+$ source aws-env/bin/activate
 ```
 
+Activate the environment in Windows:
+
+```bat
+ aws-env\Scripts\activate
```

Comment:
Should we be consistent in whether we include the shell prompt?  It’s weird to have it for Linux but not for Windows.

Reply:
We'll have to update the copy button script to ignore the shell prompt for `bat`, so it isn't included in the clipboard (like we do for `bash`/`shell`).

Reply:
I'll add it back in, should I create an issue for the copy script button?

Reply:
Or just leave it off, and I’ll take care of adding a prompt for all the windows snippets as part of updating the copy butt script. Please open an issue, and assign to me.